examples/example_batch_execution.py

```diff
@@ -0,0 +1,104 @@
+import pandas as pd
+from sklearn.preprocessing import StandardScaler
+from sklearn.svm import SVR
+
+# Import the pyWATTS pipeline and the required modules
+from pywatts.callbacks import CSVCallback, LinePlotCallback
+from pywatts.core.computation_mode import ComputationMode
+from pywatts.core.pipeline import Pipeline
+from pywatts.modules import ClockShift, LinearInterpolater, RollingRMSE, SKLearnWrapper
+
+
+# This function creates and returns the preprocessing pipeline
+def create_preprocessing_pipeline(power_scaler):
+    pipeline = Pipeline(path="../results/preprocessing")
+
+    # Deal with missing values through linear interpolation
+    imputer_power_statistics = LinearInterpolater(method="nearest", dim="time",
+                                                  name="imputer_power")(x=pipeline["scaler_power"])
+    # Scale the data using a standard SKLearn scaler
+    scale_power_statistics = power_scaler(x=imputer_power_statistics)
+
+    # Create lagged time series to later be used in the regression
+    ClockShift(lag=1)(x=scale_power_statistics)
+    ClockShift(lag=2)(x=scale_power_statistics)
+    return pipeline
+
+
+# This function creates the pipeline which we use for testing.
+# The test pipeline works on batches with one hour
+def create_test_pipeline(regressor_svr):
+    # Create test pipeline which works on a batch size of one hour.
+    pipeline = Pipeline("../results/test_pipeline", batch=pd.Timedelta("1h"))
+
+    # Add the svr regressor to the pipeline. This regressor should be called if it is not daytime
+    regressor_svr_power_statistics = regressor_svr(ClockShift=pipeline["ClockShift"],
+                                                   ClockShift_1=pipeline["ClockShift_1"],
+                                                   target=pipeline["load_power_statistics"],
+                                                   computation_mode=ComputationMode.Refit,
+                                                   callbacks=[LinePlotCallback('SVR')],
+                                                   lag=pd.Timedelta(hours=24),
+                                                   train_if=lambda x, _: len(x["ClockShift"].indexes["time"]) > 0 and
+                                                                         x["ClockShift"].indexes["time"][0].hour == 0)
+    RollingRMSE(window_size=1, window_size_unit="d")(
+        y_hat=regressor_svr_power_statistics, y=pipeline["load_power_statistics"],
+        callbacks=[LinePlotCallback('RMSE'), CSVCallback('RMSE')])
+    return pipeline
+
+
+if __name__ == "__main__":
+    # Read the data via pandas.
+    data = pd.read_csv("../data/getting_started_data.csv", parse_dates=["time"], infer_datetime_format=True,
+                       index_col="time")
+
+    # Split the data into train and test data.
+    train = data[:6000]
+    test = data[8700:]
+
+    # Create all modules which are used multiple times.
+    regressor_svr = SKLearnWrapper(module=SVR(), name="Regression")
```

Comment:
Mixed style of names (Regression, ClockShift, load_power_statistics). Would prefer a common style.